modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/workspaces/__tests__/__snapshots__/EmptyState.jsx.snap

```diff
@@ -74,7 +74,7 @@ exports[`Workspace Empty State should render 1`] = `
           90 Day Full Feature Trial
         </p>
         <a
-          class="c-link link-secondary text-decoration-underline"
```

Comment:
Hi @ilzamcmed, why were these snapshots updated in this PR?

Reply:
They were failing on master. Should I put them in another PR?